Error linking on Windows 64

Shiboken2 fails to link on Appveyor. Does anybody know what goes on here?

https://ci.appveyor.com/project/techtonik/pyside2-setup/build/1.0.31#L1048

```
...
[ 95%] Linking CXX executable shiboken2.exe
	cd C:\projects\pyside2-setup\pyside_build\py2.7-qt5.5.1-32bit-release\shiboken2\generator
	"C:\Program Files (x86)\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\shiboken2.dir --manifests  -- C:\PROGRA~2\MICROS~1.0\VC\bin\link.exe  @CMakeFiles\shiboken2.dir\objects1.rsp @C:\Users\appveyor\AppData\Local\Temp\1\nm1735.tmp
Microsoft (R) Incremental Linker Version 9.00.30729.01
 
Copyright (C) Microsoft Corporation.  All rights reserved.
 
 
 
CMakeFiles/shiboken2.dir/generator.cpp.obj CMakeFiles/shiboken2.dir/shiboken2/cppgenerator.cpp.obj CMakeFiles/shiboken2.dir/shiboken2/headergenerator.cpp.obj CMakeFiles/shiboken2.dir/shiboken2/overloaddata.cpp.obj CMakeFiles/shiboken2.dir/shiboken2/shibokengenerator.cpp.obj CMakeFiles/shiboken2.dir/shiboken2/shibokennormalize.cpp.obj CMakeFiles/shiboken2.dir/main.cpp.obj 
 
apiextractor.lib(name_compiler.cpp.obj) : error LNK2001: unresolved external symbol "__declspec(dllimport) bool __cdecl operator==(class QString const &,class QString const &)" (__imp_??8@YA_NABVQString@@0@Z)
 
apiextractor.lib(typedatabase.cpp.obj) : error LNK2001: unresolved external symbol "__declspec(dllimport) bool __cdecl operator==(class QString const &,class QString const &)" (__imp_??8@YA_NABVQString@@0@Z)
...
```

It looks like maybe Qt libraries are missing. How to check that they are ok?

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,25 +1,18 @@
 init:
-  - cmd: python --version
-  - cmd: python -c "import platform; print(platform.machine())"
+  - cmd: C:\Python27-x64\python.exe --version
+  - cmd: C:\Python27-x64\python.exe -c "import platform; print(platform.machine())"
 
 install:
   - cmd: git submodule update --init --recursive
 
 build_script:
-  - cmd: >-
-      cmake --version
-      dir
-      cd C:\Qt
-      rem curl -sLO https://download.qt.io/archive/qt/5.5/5.5.0/qt-opensource-windows-x86-msvc2010-5.5.0.exe
-      rem 7z 9.20 fails to unpack the archive properly (only 13/585Mb are unpacked)
-      rem 7z x qt-opensource-windows-x86-msvc2010-5.5.0.exe
-      
-      cd C:\projects\pyside2-setup
-      dir C:\Qt\5.5
-      dir C:\Qt\5.5\mingw492_32
-      dir C:\Qt\5.5\mingw492_32\bin
-      dir C:\OpenSSL-Win32
-      set QTDIR=C:\Qt\5.5\mingw492_32
-      set PATH=%PATH%;%QTDIR%\bin
-      qmake --version
-      python setup.py install --openssl=C:\OpenSSL-Win32
+  - cmd: cmake --version
+  - cmd: rem curl -sLO https://download.qt.io/archive/qt/5.5/5.5.1/qt-opensource-windows-x86-msvc2013-5.5.1.exe
+  - cmd: rem 7z 9.20 fails to unpack the archive properly (only 13/585Mb are unpacked)
+  - cmd: rem 7z x qt-opensource-windows-x86-msvc2013-5.5.1.exe
+  - cmd: call "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" x86_amd64
+  - cmd: set QTDIR=C:\Qt\5.5\msvc2013_64
+  - cmd: set PATH=%PATH%;%QTDIR%\bin
+  - cmd: qmake --version
+  - cmd: cd C:\projects\pyside2-setup
+  - cmd: C:\Python27-x64\python.exe setup.py install --openssl=C:\OpenSSL-Win32

